laboratory/.gitignore

```diff
@@ -0,0 +1,9 @@
+node_modules/
+/test-results/
+/playwright-report/
+/playwright/.cache/
+/test-results/
+/playwright-report/
+/playwright/.cache/
+/tests/shared/data/
+!/tests/shared/data/extension.txt
```

Comment:
Feels odd to need `!`, is it possible to refactor to do without this? Generally I prefer being able to e.g. delete a build/cache/output folder and not worry about things inside needing to persist. But not sure exactly how this is used.